LoopKit/CarbKit/CachedCarbObject+CoreDataClass.swift

```diff
@@ -38,19 +38,6 @@ class CachedCarbObject: NSManagedObject {
         }
     }
 
-    var uploadState: UploadState {
```

Comment:
Q because I don't know - how are we now keeping track of upload state/if a carb entry has been updated? Is it via the query anchors?

Reply:
Yes. The query anchors and entire Remote Data Service ecosystem now control uploads. The uploadState was deprecated (but not removed) when RDS went in place. The `externalID` is also a part of the old system, but it is still in use during a migration from the old system to new RDS system.